owners/test/index.test.js

```diff
@@ -57,7 +58,18 @@ const ownersRules = [
   new OwnersRule('dir2/dir1/dir1/OWNERS', [new UserOwner('githubuser')]),
 ];
 
-describe('owners bot', () => {
+/**
+ * Require a fresh copy of a module, clearing the require cache.
+ *
+ * @param {string} path module path.
+ * @return {*} required module.
+ */
+function _rerequire(path) {
```

Comment:
Look at how I implemented `getFixture` in my projects.

I considered doing this `delete.require`/`require` thing too, but realized that this abuses the `require` function, which is supposed to be for code, not for fixtures. You should change all the fixture-retrieving code to load it with the `fs` module and not with `require`

https://github.com/ampproject/amp-github-apps/blob/master/bundle-size/test/_test_helper.js#L28-L32

Reply:
Done.